m/main_glider_data_processing_dt.m

```diff
@@ -484,7 +534,9 @@
           seaglider_time_sensor_select = ...
             strcmp('elaps_t', {preprocessing_options.time_list.time});
           preprocessing_options.time_list(seaglider_time_sensor_select).conversion = ...
-            @(t)(t +  meta_raw.start_secs);
+              @(t)(t +  meta_raw.start_secs);
+        case 'seaexplorer' % check lat/lon integrity
+          data_raw = checkSXPositionIntegrity(data_raw, preprocessing_options);
```

Comment:
I think it would be better for you to perform the invalid position filtering by configuring the conversion function
in the position_list option in configDataPreprocessingSeaexplorer to your own version of nmea2deg,
setting the null positions to NaNs there.